feature-libs/cart/base/components/cart-shared/cart-item-list/cart-item-list.component.ts

```diff
@@ -104,9 +103,16 @@ export class CartItemListComponent implements OnInit, OnDestroy {
     );
   }
 
+  protected setItemsInternal(items: OrderEntry[], forceRerender?: boolean) {
```

Comment:
[MEDIUM]
For more semantic invocations of this function, what about making the 2nd argument an object? e.g;
```ts
method(items: OrderEntry[], options?: {forceRerender?: boolean})
```

benefits: 
- calling it is more self-explanatory, e.g. `this.method(items, {forceRerender:true})` (instead of `this.method(items, true)`)
- leaves a room allows for potentially adding _new optional_ properties to the `options` object in the future

Note: analogical suggestion applies to the method `this.rerenderChangedItems`